src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -233,6 +234,66 @@ public OrganisationsDetailResponse retrieveOrganisations() {
         return updatedOrganisationDetails;
     }
 
+    @Override
+    public OrganisationsDetailResponse retrieveOrganisations(String group) {
+        List<Organisation> retrievedOrganisations = organisationRepository.findAll();
+
+        List<Organisation> pendingOrganisations = new ArrayList<>();// = organisationRepository.findByStatus(OrganisationStatus.PENDING.name());
+        List<Organisation> activeOrganisations = new ArrayList<>();// = retrieveActiveOrganisationDetails();
+        List<Organisation> resultingOrganisations = new ArrayList<>();
+
+        retrievedOrganisations.forEach(organisation -> {
+            if (organisation.getStatus() == OrganisationStatus.ACTIVE) {
+                activeOrganisations.add(organisation);
```

Comment:
this can be done through a single loop rather than running two separate for each for pending and active orgs

Reply:
true, i will implement this